client/components/ItemActionsMenu/index.jsx

```diff
@@ -46,7 +46,7 @@ export class ItemActionsMenu extends React.Component {
         }
 
         return (
-            <button className={className} onClick={this.toggleMenu}>
+            <button className={className} onClick={this.toggleMenu} title={gettext(TOOLTIPS.actions)}>
```

Comment:
Using `gettext` with a variable will not provide translations to `.po` files.
Use `gettext` with a **static string** only

Reply:
Done, but left the constants might be useful!